lightning/src/chain/channelmonitor.rs

```diff
@@ -1110,15 +1117,26 @@ impl<Signer: Sign> ChannelMonitor<Signer> {
 			payment_hash, payment_preimage, broadcaster, fee_estimator, logger)
 	}
 
-	pub(crate) fn broadcast_latest_holder_commitment_txn<B: Deref, L: Deref>(
+	pub(crate) fn maybe_broadcast_latest_holder_commitment_txn<B: Deref, L: Deref>(
+		&self,
+		broadcaster: &B,
+		logger: &L,
+	) where
+		B::Target: BroadcasterInterface,
+		L::Target: Logger,
+	{
+		self.inner.lock().unwrap().maybe_broadcast_latest_holder_commitment_txn(broadcaster, logger)
+	}
+
+	pub(crate) fn force_broadcast_latest_holder_commitment_txn_unsafe<B: Deref, L: Deref>(
```

Comment:
Right. We talked before about adding this method so that the users still has an option to broadcast if they think they know what they are doing, but I think in that case we want it to be plain `pub`, right? Changed this in [9084457](https://github.com/lightningdevkit/rust-lightning/pull/1593/commits/9084457d580cd271a53972b1417df6a865bf6e64)

Reply:
Actually, I think you also suggested this in another comment above. I rediscovered visibility rules, lol.